src/test/functional/features/applicant1-joint.feature

```diff
@@ -295,6 +295,8 @@ Feature: Applicant 1 joint application
     Given I click "I cannot enter a UK postcode"
     And I select "Address line 1"
     And I type "MINISTRY OF JUSTICE, SEVENTH FLOOR, 102 PETTY FRANCE"
+    And I select "Town or city"
```

Comment:
is town a requirement?

Reply:
it is. I'm confused how this error has only cropped up now though